setup.cfg

```diff
@@ -22,7 +30,7 @@ spellcheck-targets = comments
 dictionaries = en_US,python,technical
 max-cognitive-complexity = 14
 max-expression-complexity = 7
-per-file-ignores = 
+per-file-ignores =
 	test_*: E800
```

Comment:
Why are we ignoring commented out code in test files?

Reply:
I think it should flagged on individual lines in the code. Let me create a separate PR for this as I expect there would be errors after removing this.